histogram/histogram_test.go

```diff
@@ -80,3 +82,56 @@ func ExampleHist_duration() {
 	// 800ms-900ms  4.55%  ▋       1
 	// 900ms-1s     9.09%  █▏      2
 }
+
+func TestHistByCustomRange(t *testing.T) {
+	Convey("Test HistByCustomRange function", t, func() {
```

Comment:
Please respect the original style of test of this package. In particular, don't bring in `goconvey` for testing. Perhaps use `testify/require` if you really want some testing helper, but not an entire new testing paradigm like `goconvey`.

Reply:
goconvey is  powerful test tools, perf goconvey import by your comment